src/routes/favorites.js

```diff
@@ -1,72 +1,70 @@
 import React from "react";
-import { useHistory } from 'react-router-dom';
+import {
+  Link,
+  useHistory
+} from 'react-router-dom';
 import {
   StudySpaceCard,
   Topbar
 } from "../components";
 
-// TODO: Read the JSON file from App.js
-var study_spaces = require("../study-spaces.json");
-var dataHtml = [];
+function handleClick() {
+  console.log("Favorites card clicked.");
+
+  // TODO: Commented this out because it was breaking the back button.
+  // history.push("/view-space");
+}
+
+/**
+ * Improvements:
+ *  - update list as soon as it's clicked instead of having to refresh the page
```

Comment:
having to refresh the page is honestly kind of awkward on a UI-perspective. maybe see suggested fix on #25 or check out `search_form.js` which updates itself WITHOUT having to refresh the page

Reply:
Created an issue (#29) for next week to figure it out tmr night.